--- libsemanage-3.10/src/conf-parse.y.orig	2026-02-02 03:01:16.000000000 +0100
+++ libsemanage-3.10/src/conf-parse.y	2026-04-23 21:21:09.023119399 +0200
@@ -401,7 +401,7 @@ static int semanage_conf_init(semanage_c
 	}
 	conf->ignoredirs = NULL;
 	conf->store_root_path = strdup("/var/lib/selinux");
-	conf->compiler_directory_path = strdup("/usr/libexec/selinux/hll");
+	conf->compiler_directory_path = strdup(LIBEXECDIR "/selinux/hll");
 	conf->policyvers = sepol_policy_kern_vers_max();
 	conf->target_platform = SEPOL_TARGET_SELINUX;
 	conf->expand_check = 1;
--- libsemanage-3.10/src/Makefile.orig	2026-02-02 03:01:16.000000000 +0100
+++ libsemanage-3.10/src/Makefile	2026-04-23 21:22:06.306142404 +0200
@@ -10,6 +10,7 @@ PKG_CONFIG ?= pkg-config
 # Installation directories.
 PREFIX ?= /usr
 LIBDIR ?= $(PREFIX)/lib
+LIBEXECDIR ?= $(PREFIX)/libexec
 INCLUDEDIR ?= $(PREFIX)/include
 SYSCONFDIR ?= /etc
 PYINC ?= $(shell $(PKG_CONFIG) --cflags $(PYPREFIX))
@@ -60,7 +61,7 @@ CFLAGS ?= -Werror -Wall -W -Wundef -Wsha
 SWIG_CFLAGS += -Wno-error -Wno-unused-but-set-variable -Wno-unused-variable -Wno-shadow \
 		-Wno-unused-parameter -Wno-missing-prototypes -Wno-missing-field-initializers
 
-override CFLAGS += -I../include -I../../libselinux/include -D_GNU_SOURCE
+override CFLAGS += -I../include -D_GNU_SOURCE -DLIBEXECDIR=\"$(LIBEXECDIR)\"
 RANLIB ?= ranlib
 
 SWIG = swig -Wall -python -o $(SWIGCOUT) -outdir ./
